src/libcrashreporter-handler/CMakeLists.txt

```diff
@@ -13,6 +13,25 @@ list(APPEND crashreporter-handler_LIBRARIES
     crashreporter-breakpad
 )
 
+if(UNIX AND NOT APPLE)
+set(CMAKE_AUTOMOC 1)
```

Comment:
You should set this per target not "globally"
set_target_properties() or something